src/sentry/integrations/slack/utils/escape.py

```diff
@@ -14,3 +16,16 @@ def escape_slack_text(txt: str | None) -> str:
     if not txt:
         return ""
     return txt.translate(translator)
+
+
+def escape_slack_markdown_text(txt: str | None) -> str:
+    """
+    Reduces runs of multiple backticks to a single backtick.
+    This prevents the "mrkdwn" code block from ending early and leaving the rest of the text out of the code block.
+    """
+    if not txt:
+        return ""
+
+    backtick_pattern = re.compile(r"`+")
```

Comment:
nit: you could probably instantiate this globally as a constant since it's not dynamic in nature